### ISSUE TITLE
Issue with Static Link Cards, Using macros

### ISSUE THREAD

**Issue by candideu** [pengx17/logseq-plugin-link-preview#127]
Hello!

I'm using the latest version of your plugin on the latest version of Logseq on Windows 10. I really love using the Link Cards function, but the static link cards haven't worked for me for as long as I've used this plugin:

![image](https://user-images.githubusercontent.com/55474996/227735507-d71281be-b867-4253-9d68-1e6b9321566e.png)

I tend to stick to the regular link card slash command, but the issue is that on mobile, the links just don't appear:

**Desktop**
![image](https://user-images.githubusercontent.com/55474996/227735589-957058a0-fdcf-4caf-9729-36c042d6ed91.png)

**Mobile**

https://user-images.githubusercontent.com/55474996/227735758-5ffff762-e5a5-424c-bd30-5077a5bcd74d.mp4

I wonder if there's another way of managing the link cards to ensure that the links appear, even on mobile / web app / any version that doesn't have the plugin.

**Comment by pengx17** [comment]
The macro requires the plugin system, but plugin is not being supported on Mobile as long as I know. :(

**Comment by candidexmedia** [comment]
Would it be possible to create a macro that works regardless of whether the plugin is installed or not? So once someone creates a link card, it displays as it should with the plugin enabled, but when the plugin isn't present, it just displays a link?

Macros seem to work on mobile and desktop.

**Comment by candideu** [comment]
Would it be possible to create a macro that works regardless of whether the plugin is installed or not? So once someone creates a link card, it displays as it should with the plugin enabled, but when the plugin isn't present, it just displays a link?

Macros seem to work on mobile and desktop.

**Comment by pengx17** [comment]
Yeah, Macros does work on mobile, but the `{{link-preview ...}}` macro requires the plugin to work.
